### ISSUE TITLE
some questions about the code

### ISSUE THREAD

**Issue by mzj-ustc** [AdityaJoglekar/DMF-TONN#1]
I ran your code, changed tensorflow to pytorch, adjusted the learning rate, and then we found that the loss can be reduced to about -71 instead of -0.005, which leads to the error of the whole calculation. Is there any problem with this version of the code?
